docs/tutorials/blockheads.rst

```diff
@@ -0,0 +1,90 @@
4"},{"id":"288","price":"739912.44947533","old_price":"697998.17778503"},{"id":"289","price":"365576.51480898","old_price":"321752.36520955"},{"id":"290","price":"348005.88207467","old_price":"312578.23264018"},{"id":"291","price":"10946.818045413","old_price":"10976.781884879"},{"id":"292","price":"16737.705966991","old_price":"16981.782886612"},{"id":"293","price":"9403.3568534338","old_price":"9074.6010472562"},{"id":"294","price":"9371.9444470195","old_price":"9073.5708253308"},{"id":"295","price":"12.372771341069","old_price":"10.660934664874"},{"id":"1024","price":"0.1333739379752","old_price":"0.12177429039489"},{"id":"1026","price":"11.043503766122","old_price":"11.70922669034"},{"id":"1028","price":"3.0024169146676","old_price":"2.7867898824208"},{"id":"1030","price":"23.601363916926","old_price":"22.665800003435"},{"id":"1036","price":"11.075555028821","old_price":"13.037483563067"},{"id":"1038","price":"1621.4729384406","old_price":"1529.5470722464"},{"id":"1042","price":"15.36189882995","old_price":"16.079559623694"},{"id":"1043","price":"8.4727426099765","old_price":"8.0380449991327"},{"id":"1045","price":"8505.3879024171","old_price":"8570.9992523412"},{"id":"1048","price":"1.6385279853728","old_price":"1.5602288882267"},{"id":"1049","price":"6.3319813752839","old_price":"6.1150467907364"},{"id":"1051","price":"2.6407522360647","old_price":"2.6124787973121"},{"id":"1054","price":"1549.3090847174","old_price":"1441.8043610669"},{"id":"1057","price":"58.636369366839","old_price":"60.776850361544"},{"id":"1060","price":"4.9075833665728","old_price":"5.102553932843"},{"id":"1062","price":"87.749303545728","old_price":"79.863847806401"},{"id":"1063","price":"146.20246598437","old_price":"152.53525056421"},{"id":"1065","price":"1445.194108109","old_price":"1318.7889584147"},{"id":"1066","price":"75.567939344601","old_price":"79.888522034962"},{"id":"1067","price":"88.539363058084","old_price":"73.761783158688"},{"id":"1068","price":"123.69608473135","old_price":"103.41878831344"},{"id":"1069","price":"1282.5178346444","old_price":"1239.5648444443"},{"id":"1070","price":"1693.1317681708","old_price":"1641.7545236235"},{"id":"1072","price":"1601.4918195748","old_price":"1674.9368675029"},{"id":"1075","price":"49.325749587432","old_price":"44.126504990343"},{"id":"1076","price":"94.986025144929","old_price":"83.039375097592"},{"id":"1077","price":"6105.5953654217","old_price":"5762.0530979032"},{"id":"1082","price":"4878.6501378555","old_price":"4416.912543811"},{"id":"1083","price":"3753.4879512545","old_price":"4063.4148323552"},{"id":"1087","price":"517.55244496814","old_price":"635.30711575861"},{"id":"1088","price":"3220.3008809241","old_price":"3330.9123007421"},{"id":"1089","price":"81034.756544755","old_price":"85016.6041781"},{"id":"1090","price":"8011.3564639592","old_price":"8003.6717305029"},{"id":"1091","price":"868685.40593135","old_price":"763117.08163949"},{"id":"1094","price":"8.1057218746355","old_price":"9.8404860850315"}]
```

Comment:
Would it make sense to shorten this a bit. Github displays a scrolling box - and I guess almost nobody will scroll to the very right of it anyway...